lib/cylc/task_state.py

```diff
@@ -398,7 +390,7 @@ def _set_state(self, status):
             message += " (%s)" % self.hold_swap
         LOG.debug(message, itask=self.identity)
 
-    def is_greater_than(self, status):
+    def is_gt(self, status):
```

Comment:
Why `status_leq` and `status_geq` are module functions, while this is a method?

Reply:
Originally only the method was used, then this branch needed the operands to be the other way around.  I'll get rid of the method...